lua/autorun/jmod_legacy_client.lua

```diff
@@ -231,7 +231,7 @@ if(CLIENT)then
 	local function JackyClientHeadcrabRemoval(data)
 		local Pos=data:ReadVector()
 		timer.Simple(.01,function()
-			for key,rag in pairs(ents.FindInSphere(Pos,90))do
+			for key,rag in ipairs(ents.FindInSphere(Pos,90))do
```

Comment:
why ipairs

Reply:
I'm just gonna reply what I think is going on.
For all of these built in funcs that return a table of entities and such they are 99% of the time returning indexed tables
and ipairs is supposed to be faster at indexed tables than pairs.

Reply:
AdventureBoots is right